locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -3984,13 +4117,16 @@ msgstr ""
 "href=\"%(email_url)s\">fordern Sie eine neue Bestätigungsemail an</a>."
 
 #: meinberlin/templates/account/login.html:9
-#, python-format
-msgid ""
-"If you have not created an account yet, then please <a href=\"%(signup_url)s"
-"\">register</a> first."
+msgid "If you have not created an account yet, then please "
+msgstr ""
+
+#: meinberlin/templates/account/login.html:9
+msgid "register"
+msgstr ""
+
+#: meinberlin/templates/account/login.html:9
+msgid " first."
```

Comment:
The seem to be some translations missing. Mind adding them in this PR so we don't forget?

Reply:
If that's planned for later - never mind.

Reply:
Öhm, there's something else that went wrong here. The string in login and register is formatted quite strangely. Will check!